lib/client/file-client/FileClient.js

```diff
@@ -25,19 +36,23 @@ client.init = function() {
     deferredReady.resolve();
   }
 
-  client.initPromise = q.all([deferredFhinit.promise, deferredReady.promise]);
-  return client.initPromise;
+  this.initPromise = q.all([deferredFhinit.promise, deferredReady.promise]);
+  return this.initPromise;
 };
 
-client.uploadDataUrl = function(userId, dataUrl) {
+
+/***
+ * List files for particular user. If user parameter is not provided returns list of all files
+ *
+ * @param userId
+ */
+FileClient.prototype.list = function(userId) {
+  var url = arguments.length === 0 ? config.apiPath + '/all'
+    : config.apiPath + '/owner/' + userId;
```

Comment:
wouldn't `var url = !userId ? config.apiPath + '/all'  : config.apiPath + '/owner/' + userId;` easier to read?